### ISSUE TITLE
NTR: Subependymal nodules

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4059]
Synonyms: HP:0009716 
; HPO ID: NA 
; Alternative ID: Cerebral hamartomata 
; Son of_1: NA 
; Son of_2: NA 
; Son of_3: NA 
; Son of_4: Small nodular masses which originate in the subependymal region of the lateral ventricles and protrude into the ventricular cavity. They may represent subependymal hamartomas of tuberous sclerosis or nodular heterotopia of grey matter. 
; Textual definition: NA 
; PMID to support changes:  
; label: NA 
; Curator: 191100 613254  
; OMIM -disease: 605284 191092 
; OMIM -gene

**Comment by pnrobinson** [comment]
Term already exists
https://hpo.jax.org/app/browse/term/HP:0009716
I will transfer the annotation request to the HPO annot repo
